components/admin_console/permission_schemes_settings/permission_system_scheme_settings/permission_system_scheme_settings.jsx

```diff
@@ -51,6 +53,14 @@ export default class PermissionSystemSchemeSettings extends React.Component {
         this.rolesNeeded = ['system_admin', 'system_user', 'team_admin', 'team_user', 'channel_admin', 'channel_user'];
     }
 
+    excludedPermissions = [
```

Comment:
Would it make sense to make `excludedPermissions` a constant?

Reply:
Yes, probably makes sense, I'll do the change :)

Reply:
Done :)